lib/spree/core/permalinks.rb

```diff
@@ -70,6 +62,3 @@ def save_permalink(permalink_value = to_param)
     end
   end
 end
-
-ActiveRecord::Base.include(Spree::Core::Permalinks)
-ActiveRecord::Relation.include(Spree::Core::Permalinks)
```

Comment:
I wonder if not including it in `ActiveRecord::Relation` will have any consequences..?

Reply:
I wondered that, too. But I think that the reason for this were the find_by_param methods. Spree could do something like `Product.in_stock.find_by_param`. But we don't use those any more and I removed them. I couldn't find any usage of the other methods on scopes.